recipe/meta.yaml

```diff
@@ -33,6 +35,9 @@ requirements:
     - mkl-service
     - cxx-compiler  # [not win]
     - m2w64-toolchain  # [win]
```

Comment:
I think these dependencies should be in the aesara feedstock, not this one.

Reply:
Hmm, maybe it's best to keep them for this version and try to fix in Aesara soon afterwards? After removing this we'd then ≤-pin Aesara to the packages which are fixed?

BTW, I don't see Aesara as a direct dependency here. Is it indirect?

Reply:
`theano-pymc` was renamed to `aesara`, but pymc3 stable does not yet depend on it. So we might need to wait until 1. aesara has a reliable compile chain dependencies, and 2. we have a pymc3 release that depends on aesara instead of theano-pymc.